metadata-ingestion/examples/recipes/neo4j_to_datahub.dhub.yaml

```diff
@@ -0,0 +1,15 @@
+source:
```

Comment:
Can remove this recipe file. We already have one above. If not, this needs to be kept in sync with earlier changes.